app/gui/src/controller/ide/desktop.rs

```diff
@@ -38,12 +38,13 @@ const UNNAMED_PROJECT_NAME: &str = "Unnamed";
 #[derive(Clone, CloneRef, Derivative)]
 #[derivative(Debug)]
 pub struct Handle {
-    current_project:      Rc<CloneCell<Option<model::Project>>>,
+    current_project: Rc<CloneCell<Option<model::Project>>>,
     #[derivative(Debug = "ignore")]
-    project_manager:      Rc<dyn project_manager::API>,
+    project_manager: Rc<dyn project_manager::API>,
     status_notifications: StatusNotificationPublisher,
-    parser:               Parser,
-    notifications:        notification::Publisher<Notification>,
+    parser: Parser,
+    notifications: notification::Publisher<Notification>,
+    private_component_browser_entries_visibility_flag: Rc<Cell<bool>>,
```

Comment:
I would rename to `component_browser_private_entries_visibility`. The entries are private, not CB.